tsconfig.base.json

```
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    "module": "CommonJS",
+    "target": "ES2022",
+    "sourceMap": true,
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "noImplicitThis": true,
+    "noImplicitAny": true,
+    "strictNullChecks": true,
+    "allowSyntheticDefaultImports": true,
+    "forceConsistentCasingInFileNames": true,
+    "downlevelIteration": true,
+    "importHelpers": true,
+    "allowJs": true,
+    "baseUrl": ".",
+    "paths": {
+      "@romcal/rite-roman1962": ["./rites/roman1962"],
+      "@romcal/rite-roman1969": ["./rites/roman1969"],
+      "@romcal/config": ["./packages/config"],
+      "@romcal/easter": ["./packages/easter"],
+      "@romcal/package.json": ["./package.json"]
```

could we consider not having the `@romcal/` prefix, as these will conflict with future package names?

(and add some others i mentioned elsewhere)

i.e.
```suggestion
      "@rite-roman1962": ["./rites/roman1962"],
      "@rite-roman1969": ["./rites/roman1969"],
      "@config": ["./packages/config"],
      "@easter": ["./packages/easter"],
      "@package.json": ["./package.json"]
```

---

I like keeping the `@romcal/` prefix to clearly identify that these are romcal-specific packages. And I think that if some of these packages are published someday, we can always adjust the naming without any issues.

Or maybe we should use a different prefix to identify the romcal packages that are published and those that are not.

---

so, problem is that these aren't packages, these are in-type aliases, and the actual packages can be referenced directly in the system by their full package name. I think what you're doing makes sense, but now how you're doing it.

---

i.e. `@romcal/config` shouldn't be usable outside of this and this might lead someone to believe this will also be a separate stand-alone released package... but given that it's purely internal to the monorepo, this should be an internal config reference, not something I could consume in another project.

---

The issue here is that, packages like `@romcal/rite-roman1962` are actual NPM packages (they have their own `package.json` files), they're just not published on npmjs.org. But for everything to work well in the monorepo, I also had to define them as aliases. Also, the configuration varies slightly depending on the package manager (npm, yarn, pnpm...), and I haven't fully explored how to optimize the setup here (so inputs are welcome here).

For now, I’ve renamed the @romcal/* packages to @internal/* while these packages remain unpublished. If some of them do get published, then we can rename them back to @romcal/*, which should reduce the risk of confusion.